frontend/src/scenes/funnels/funnelLogic.ts

```diff
@@ -24,21 +25,45 @@ import { trendsLogic } from 'scenes/trends/trendsLogic'
 import { FunnelStepReference } from 'scenes/insights/InsightTabs/FunnelTab/FunnelStepReferencePicker'
 import { eventDefinitionsModel } from '~/models/eventDefinitionsModel'
 
+function aggregateBreakdownResult({ result: breakdownList, ...apiResponse }: FunnelResultWithBreakdown): FunnelResult {
+    let result: FunnelStep[] = []
+    if (breakdownList.length) {
+        result = breakdownList[0].map((step, i) => ({
+            ...step,
+            breakdown_value: step.breakdown as any,
+            count: breakdownList.reduce((total, breakdownSteps) => (total += breakdownSteps[i].count), 0),
```

Comment:
Use of `reduce` here is to compensate for the fact that the API response does not include aggregated / hierarchical data. Have talked with @EDsCODE about changing the shape of the response value — I can possibly even take that on myself in the future